### ISSUE TITLE
Screen reader (Skjermleser) reads progress indicator (framdriftindikator) as a date

### ISSUE THREAD

**Issue by xmrsa** [Altinn/app-frontend-react#1621]
### Description of the bug

Using a screen reader (Edge's inbuilt screen reader ) I did not expect to hear "femte november" when it described the text shown in the progress bar. The correct text should have been something like "side 5 av 11".

### Steps To Reproduce

1. Start a screen reader in an app with a progress indicator.
2. Listen as it navigates to the round process indicator symbol.
![image_480](https://github.com/Altinn/altinn-studio/assets/42466346/fbac23f6-b480-451b-ae45-6b1522782c36)

### Additional Information

Ivar Nesje sier vi trenger en **aria-label** for å fikse dette problemet https://altinn.slack.com/archives/C02EVE4RU82/p1698851531930709?thread_ts=1698851044.548469&cid=C02EVE4RU82

**Comment by xmrsa** [comment]
This might be related to this: https://github.com/Altinn/app-frontend-react/issues/1622

**Comment by xmrsa** [comment]
This might be an issue only related to how Edge's screen reader is implemented, because there already is an aria-label on this element. This screenshot is from Slack (https://altinn.slack.com/archives/C02EVE4RU82/p1698913703359999?thread_ts=1698851044.548469&cid=C02EVE4RU82)
![image](https://github.com/Altinn/app-frontend-react/assets/42466346/c6cc94fd-9ea1-495b-84b3-a50eb80a705f)
